image-search/Models/Album.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Album.swift
+//  image-search
+//
+//  Created by Renato Vieira on 08/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+
+struct SearchImages: Codable{
```

Comment:
Name of file and structs are different. Try to keep the same.

Reply:
Done